### PR TITLE
Update the package and testfile

### PR DESCRIPTION
**Pull request by howardpen9**
1/ Using the  "@ton-community/sandbox",  "@ton-community/test-utils"

2/ Delete the part of file in @tact-lang/emulator

### CODE REVIEW

File: package.json
Comment (anton-trunov) [inline package.json:1]: This file has some unresolved merge conflicts

File: sources/contract.spec.ts
Comment (anton-trunov) [inline sources/contract.spec.ts:1]: This file also has some unresolved merge conflicts

**Comment by anton-trunov** [comment]
Hey @howardpen9, thanks for the PR, could you also clarify the purpose of this PR?

**Comment by novusnota** [comment]
To me, it seems like @howardpen9 here tried to move from the `@tact-lang/emulator` over to `@ton-community/sandbox` and `@ton-community/test-utils`. The latter two are now hosted under `@ton-org/sandbox` and `@ton-org/test-utils`, and they are much more maintained and widely used (in Blueprint, for example), than the `@tact-lang/emulator`

I think we should open an issue for this case, as to prefer using `@ton-org/sandbox` over the `@tact-lang/emulator`, what do you think, @anton-trunov?

**Comment by anton-trunov** [comment]
sure, let's open an issue

**Pull request closed without merging**